report_pre_thesis/src/index.tex

```diff
@@ -70,9 +72,32 @@ \subsection{Problem Statement Proposal}
 % This raises a critical question: How can the MOC model be refined to enhance its accuracy and robustness, ensuring more reliable predictions that can better support the objectives of the MSL mission?
 % \subsection{comment-end}
 
-In this work, we aim to identify potential limitations and areas of improvement in the Multivariate Oxide Composition model's predictive accuracy and robustness in predicting the composition of major oxides in Martian geological samples using Earth-based calibration data.
+\section{Definition}
+Given a set $E$ of elements detectable using LIBS, and considering \(\mathbb{P}(E)\) as the power set of $E$ which represents all possible combinations of these elements, we define \(K \subseteq \mathbb{P}(E) \) as the set of compositions of these elements that can be subjected to LIBS. Additionally, let $S$ be the set of all possible spectra emitted from these compositions.
+
+\begin{definition}
+    A LIBS data set $L$ for Martian geological samples is a subset of \(K' \times S \) where \(K' \subseteq K \), defined as \(L = \{(k_1, s_1 , (k_2 , s_2)) \dots (k_m , s_m) \} \). Each pair $(k_i , s_i)$ in $L$ represents the spectral data for an individual Martian geological sample, where $k_i$ is the sample's composition and $s_i$ is its emitted spectrum produced via laser-induced breakdown.
+\end{definition}
+
+\begin{definition}
+    A Calibration Set $C$ is a subset of \(K' \times S \) where \(K' \subseteq K \), defined as \(C = \{(k_1, s_1 , (k_2 , s_2)) \dots (k_n , s_n)\} \). Each pair $(k_i , s_i)$ in $C$ represents a known composition \(k_i \in K' \) from Earth's rock standards and its corresponding emitted spectrum \(s_i \in S \). Additionally, denoting the set of compositions associated with $L$ as $K_L$, then \( \forall(k,s) \in C, k \notin K_L \).
+\end{definition}
```

Comment:
Well done on formulating this mathematically, @Ivikhostrup.

What's the reasoning behind the following?
> Additionally, denoting the set of compositions associated with $L$ as $K_L$, then \( \forall(k,s) \in C, k \notin K_L \).

I think, after this section, it would be interesting to write more about the actual predictions: how they're derived, from what, etc.
However, an issue with starting entirely from scratch, as we do here, is that we'd need to also describe the entire preprocessing stage in mathematical terms. And we'd need to do so before we can even get to the predictions.

We'll need to describe the exact data that the model takes in. Perhaps we can skip the preprocessing stage's mathematical description and just say:
$C$ and $L$ undergo various preprocessing steps to normalize and adjust the data such that the various noisy components have been removed. The output of this process is data in the cleaned, calibrated spectra format, which follows the same format as previously described, except the data has been preprocessed. The primary difference is the intensity measure is now in units of photons per channel in the spectral intensity data.

And then something like:
Given $(k_i , s_i)$ in $C$, a calibration model can be trained on $s_i$ to predict $k_i$.

Where you could say that the PLS-SM system uses multiple such models, each trained on a subset of each $s_i$ where the $s_i$ is partitioned into groups given its wavelengths. Then they become 'experts' at predicting compositions in that range.

But perhaps we lack some definition of $s_i$, because it would perhaps be a matrix of wavelengths associated with intensities? In which case the symbol isn't representative by convention.

Thoughts?